acclimatise/cli_types.py

```diff
@@ -74,7 +74,19 @@ class CliBoolean(CliType):
 
 
 @dataclass(unsafe_hash=True)
-class CliDir(CliType):
+class CliFileSystemType(CliType):
+    """
+    Takes a directory / file path
+    """
+
+    output: bool = None
```

Comment:
I think this should default to `False`. It doesn't make sense to have an "unknown" status because the downstream converters will get confused by this.